web/libs/editor/src/components/TopBar/TopBar.jsx

```diff
@@ -77,8 +80,8 @@ export const TopBar = observer(({ store }) => {
       ) : (
         <>
           <Elem name="group">
-            <CurrentTask store={store} />
-            {!isViewAll && (
+            {!isBulkMode && <CurrentTask store={store} />}
+            {!isViewAll && !isBulkMode && (
```

Comment:
Is this even possible to reach here with `isBulkMode` true given the check at the top of the component returns null in that case? And if it cannot be true, it is presumed only false here.

Reply:
Yep, we can have feature flag `FF_DEV_3873` turned off.
I mean it shouldn't be a usual case but for some reason I thought it's better to cover.
Although i'm not sure right now we cover all other functionality with `FF_DEV_3873` turned off, so maybe it's better to remove these 2 conditions at all. 🤔

Reply:
I have checked that. This conditions are covered by the test https://github.com/HumanSignal/label-studio/pull/6203/files#diff-15e1d28fc5b0c407a694fe6996ee68a73faa93b0ea1aa5d47bec1dd585ef82c8R146-R158
So I would say it's better to keep them there till we remove the feature flag about old ui.